pom.xml

```diff
@@ -67,7 +67,7 @@
 		<slf4j-api.version>1.7.25</slf4j-api.version>
 
 		<bouncycastle.version>1.60</bouncycastle.version>
-		<junixsocket.version>2.0.4</junixsocket.version>
+		<junixsocket.version>2.1.0</junixsocket.version>
```

Comment:
```suggestion
		<junixsocket.version>2.1.2</junixsocket.version>
```
Does this by chance make the failure go away?

Reply:
let's try

Reply:
Please update to 2.2.0 (the latest version)